src/testhost.x86/TestHostTraceListener.cs

```diff
@@ -32,6 +32,7 @@ public static void Setup()
 
         EqtTrace.Verbose("TestPlatformTraceListener.Setup: Added test platform trace listener.");
 
+        // REVIEW ME: Shall we remove this fix?
```

Comment:
As our testhost min version will be netcoreapp3.1 I believe this hack/fix can be safely removed.